### DISCUSSION TITLE
It would be awesome to export RPC client code

### DISCUSSION THREAD

**Discussion by ajubin** [honojs/hono#3741] [General]
I'm in a monorepo and I'm using generated client as explained in the doc:https://hono.dev/docs/guides/rpc#using-rpc-with-larger-applications

However I still need to import the client in my frontend from my backend.

What could be awesome is to generate a client with minimal dependency that could be used by other projects (if I don't want to use a monorepo for instance).

An inspiration: in the @tanstack/react-router where a `routeTree.gen.ts` file is generated.

**Comment by EdamAme-x** [comment]
I am interested in this.

It can be provide as `hono/build`.
https://github.com/honojs/hono/issues/3659

**Reply by EdamAme-x** [reply]
Also, if this can be achieved, it would be possible to automatically generate api clients like octokit and discordjs.

I am very interested in this.
